.github/mergify.yml

```diff
@@ -19,6 +19,8 @@ pull_request_rules:
       - label=send-it
       - author!=@libp2p/rust-libp2p-maintainers
       - author!=dependabot[bot]
+      # Do not remove approvals if last commit was a merge of master into the branch, required for the PR to be merged.
+      - commits[-1].author!=@libp2p/rust-libp2p-maintainers
```

Comment:
Nice! Ran into that few times

Reply:
yeah was going to mention it to you, glad you went ahead and noticed it ahah